app/models/annotations/task.rb

```diff
@@ -49,10 +49,6 @@ def status
     self.last_task_status
   end
 
-  def project
-    self&.annotated&.project
-  end
-
   def team
```

Comment:
move this `:team` method to `AnnotationBase`

Reply:
Done